context/app/static/js/components/genes/hooks.ts

```diff
@@ -0,0 +1,82 @@
+import { useMemo } from 'react';
+import useSWR from 'swr';
+
+import { fetcher } from 'js/helpers/swr';
+import { useGenePageContext } from './GenePageContext';
+
+const GeneApiURLs = {
+  detailURL(geneId: string) {
+    return `https://ontology.api.hubmapconsortium.org/gene/${geneId}`;
```

Comment:
Pull the api into the conf?

Reply:
Done! I originally hadn't done this since there didn't seem to be non-prod endpoints listed on the smart-api page, but it's a good practice to follow regardless.